public/app/features/auth-config/utils/url.ts

```diff
@@ -4,3 +4,12 @@ import { AuthProviderInfo } from '../types';
 export function getProviderUrl(provider: AuthProviderInfo) {
   return BASE_PATH + (provider.configPath || provider.id);
 }
+
+export const isUrlValid = (url: string): boolean => {
+  try {
+    const parsedUrl = new URL(url);
+    return parsedUrl.protocol.includes('http');
```

Comment:
I'm not sure whether we should be very strict here. I cannot think of any scenarios where this condition might block legit urls, but who knows. I would just leave the `new URL(url)` for validation on client side, but up to you.

Reply:
I have the same on the backend as well, so I'd prefer to leave it like this and make sure that scheme is either http or https.